composer.json

```diff
@@ -104,7 +110,8 @@
 		"use-github-api": false,
 		"sort-packages": true,
 		"allow-plugins": {
-			"dealerdirect/phpcodesniffer-composer-installer": true
+			"dealerdirect/phpcodesniffer-composer-installer": true,
+			"roots/wordpress-core-installer": true
```

Comment:
Is this needed, or is it left over from some earlier attempt where you had WorDBless here in the root composer.json?